docs/fsharp/style-guide/formatting.md

```diff
@@ -803,7 +814,7 @@ let function1 arg1 arg2 arg3 arg4 =
     arg3 + arg4
 ```
 
-### Formatting pipeline operators
+### Formatting pipeline operators or mutable setters
```

Comment:
```suggestion
### Formatting pipeline operators or assignments
```

Reply:
maybe `re-assignments` is better? to distinguish from normal constant assignments

Reply:
> constant assignments

I may very well be wrong, but AFAIK in F# `let x = 2` is a declaration(?), not an assignment, and only `<-` is an assignment.

Reply:
the fact that in f# you cannot declare without assigning doesn't make it not be an assignment IMO, that's why I didn't prefer this word (some people call it "symbol binding" too), but anyway we're bikeshedding here...

Reply:
Yep, I'm comfortable with "mutable setters". Maybe "mutable assignments" is the best option?

```suggestion
### Formatting pipeline operators or mutable assignments
```

As you say, we're bike-shedding here.

Reply:
Though are mutable setters/assignments related to pipeline operators in terms of formatting? It doesn't seem that way to me, so perhaps it should be its own section.